lobby/src/main/java/org/triplea/lobby/server/ModeratorController.java

```diff
@@ -119,6 +91,12 @@ private void banMac(
             banExpires == null ? "forever" : banExpires.toString()));
   }
 
+  private void assertUserIsAdmin() {
+    if (!isAdmin()) {
+      throw new IllegalStateException("Not an admin");
+    }
```

Comment:
What about using `Precoditions#checkState`?
Depending on how often this method is used, it might be ok to inline this guava precondition